src/bundle/Resources/views/form_fields.html.twig

```diff
@@ -141,3 +141,43 @@
         {{ block('form_widget') }}
     </div>
 {%- endblock -%}
+
+{% block checkbox_radio_label -%}
```

Comment:
Is it copied from Symfony? What was changed here exactly? I'm against duplicating the code, maybe we can somehow extend Symfony's template or use different approach.

Reply:
I added: `{%- if label is not same as(false) -%}` (code rendering label here) `{%- endif -%}`. 
Now, if label is set to false it is rendered with no text what is breaking tables. It was fixed by overwriting `.form-check-input` but this again is breaking checkboxes and radioboxes.

Reply:
Hmm, maybe the other approach would be to use different template for checkboxes in tables, what do you think?